winter/web/exceptions/problem_handling.py

```diff
@@ -0,0 +1,96 @@
+import re
+from typing import Dict
+from typing import Type
+
+import dataclasses
+from rest_framework.request import Request
+
+from winter.core import Component
+from .exception_handler_generator import ExceptionHandlerGenerator
+from .exception_mapper import ExceptionMapper
+from .handlers import ExceptionHandler
+from .handlers import exception_handlers_registry
+from .problem_annotation import ProblemAnnotation
+from ...core.utils import camel_to_human
```

Comment:
Please import it globally from `winter.core`. Since it's going to be a separate package anyway.